rs/nervous_system/tools/release-runscript/src/main.rs

```diff
@@ -0,0 +1,228 @@
+use colored::*;
+use std::io::{self, Write};
+
+struct Step {
+    title: &'static str,
+    description: &'static str,
+}
+
+fn main() {
+    let steps = vec![
+        Step {
+            title: "Pick Release Candidate Commit",
+            description: "1. Go to https://github.com/dfinity/ic/actions/workflows/ci-main.yml?query=branch%3Amaster+event%3Apush+is%3Asuccess
```

Comment:
We currently have a tool for this, I think it's a bash script in our nns-tools to find the most recent release with artifacts.  

list-new-commits.sh also will tell you which version it was using (with prebuilt artifacts).

Reply:
Done!